scripts/core/superdesk-api.d.ts

```diff
@@ -236,7 +237,8 @@ declare module 'superdesk-api' {
         broadcast: any;
         flags: any;
         source: string;
-        correction_sequence: any;
+        correction_sequence: number;
+        rewrite_sequence: number;
```

Comment:
Are there any docs on what do these do? It doesn't have to be in this repo, a link would work fine too.